systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaConnectS2IUtils.java

```diff
@@ -29,4 +29,15 @@ public static void waitForConnectS2IStatus(String name, String status) {
             () -> Crds.kafkaConnectS2iOperation(kubeClient().getClient()).inNamespace(kubeClient().getNamespace()).withName(name).get().getStatus().getConditions().get(0).getType().equals(status));
         LOGGER.info("Kafka Connect S2I {} is in desired state: {}", name, status);
     }
+
+    public static void waitForRebalancingDone(String name) {
+        LOGGER.info("Waiting for Kafka Connect S2I {} to rebalance", name);
+        TestUtils.waitFor("Kafka Connect S2I rebalancing", Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, Constants.TIMEOUT_FOR_RESOURCE_READINESS,
+            () -> {
+                String connect = kubeClient().listPodNames("strimzi.io/kind", "KafkaConnectS2I").get(0);
+                String log = kubeClient().logs(connect);
+                return (log.length() - log.replace("Finished starting connectors and tasks", "").length()) / "Finished starting connectors and tasks".length() == 2;
```

Comment:
This line is the count of a substring in the log string.

Reply:
You should add a comment in the code to explain that. 

And why the magic number 2?

Reply:
I do not remember exactly, but when the cluster is started, this message is in the logs. Then we lose one of the replicas and after rebalancing happened, the message is printed again.